Week2/helloWorld.js

```diff
@@ -0,0 +1,6 @@
+console.log('------ 1 ------');
+console.log('Hello World');
+console.log('Merhaba Dunya');
+console.log('Hallo Wereld');
+console.log('こんにちは世界');
+console.log('\n');
```

Comment:
Nice, but can you also add which language each `console.log` is.

Like this:

```
console.log('Hello World'); //English
```

Reply:
Hello Marcia. Thank you very much for the feedback and your time. I made the corrections as you suggested and updated my homework.

Reply:
Ups, the changes I made for this file is done by the commit with the message "updating helloWorld.js". Sorry for the confusion, still trying get used to Git.